pkg/apis/kube-scheduler/v1alpha1/types.go

```diff
@@ -23,7 +23,15 @@ type KubeScheduler struct {
 
 type KubeSchedulerSpec struct {
 	// Fill me
+	Kubeconfig string `json:"kubeconfig"`
```

Comment:
i think we dont need this as scheduler will run in a pod, and will have access via services accounts and rbac.

Reply:
Yeah, that makes sense. I will remove it.